modules/ROOT/pages/troubleshoot-guide.adoc

```diff
@@ -6,6 +6,47 @@ endif::[]
 This topic describes common errors and steps to resolve them when installing Anypoint Runtime
 Fabric.
 
+== Troubleshoot Network Connectivity Using rtfctl
+
+Every Anypoint Runtime Fabric cluster requires connectivity with Anypoint control plane, and any interference with connectivity can limit functionality, resulting in application deployment failures or degraded status in Anypoint Runtime Manager.
+
+=== Obtain a Full Assessment
```

Comment:
This section is more of a first-level diagnosis of the health of Runtime Fabric. It should be the first thing our customers do when they suspect an issue.

What I think would make sense is if we move this to its own section above the "Troubleshoot Network Connectivity" section.

Reply:
Got it - thanks!